scripts/studyarea_to_WKT/studyarea_to_WKT.yml

```diff
@@ -0,0 +1,27 @@
+script: studyarea_to_WKT.R
+description: "This script converts the geometry of the study area into a well-known text (WKT) representation."
+external_link: https://github.com/GEO-BON/biab-2.0
+inputs:
+  studyarea_path:
+    label: studyarea_path
+    description: Text string representing the file path to the spatial file containing the spatial boundaries pertaining to the study area. Accepted file types include shapefile (.shp), raster (.tif/GeoTIFF) or  GeoPackage (.gpkg). This path specifies the complete path location of the file within the file system and is utilized for identifying and accessing the spatial boundaries contained within the file.
+    type: text
+    example: "/scripts/studyarea_to_WKT/input/Arauca.gpkg"
+  studyarea_epsg:
+    label: studyarea_epsg
+    description: Numerical value referring to the EPSG code (European Petroleum Survey Group) associated with the spatial reference system that will be used as a reference for the study area. This numerical value specifies the projection and geodetic datum used to define the coordinates and spatial representation of the data in the study area. For further information on coordinate systems and EPSG codes, you can access the official database on the EPSG website at https://epsg.org/home.html. The website provides documentation, resources, and tools for searching and understanding the EPSG codes used in various geospatial contexts.
+    type: int
+    example: 3395
+outputs:
+  val_wkt_path:
+    label: val_wkt_path
+    description: Text string referring to file path of the generated WKT file (.txt) that represents the boundaries of the study area. The WKT (Well-Known Text) file is a text-based format that describes the geometry of the boundaries of the study area. The structure of the WKT file includes a geometry type label (MULTIPOLYGON) associated with the coordinate values of the vertices that define the boundaries of the study area mask.
+    type: text
```

Comment:
Should find appropriate mime type for WKT file. I found this : https://gis.stackexchange.com/questions/424129/mime-type-for-wkt?newreg=a1565fe79e0b43a5ac8ce1fe691c163f
However, text/plain is confusing in the context of this app. text/wkt would make much more sense...
Once we reach consensus, we should adjust the corresponding description.

Reply:
change to text/wkt,  i think the description y correct, don't need anything else